crates/meta/src/store/parts.rs

```diff
@@ -384,12 +384,34 @@ impl<'a> PartStore<'a> {
         Ok(())
     }
 
-    pub fn clear(&self)->MetaResult<()>{
-        //FIXME this clear stupid clear parts in the store
-        // self.tree_parts.clear().map_err(|_|MetaError::EntityDelError)?;
-        // self.tree_prids.clear().map_err(|_|MetaError::EntityDelError)?;
-        // self.tree_part_size.clear().map_err(|_|MetaError::EntityDelError)?;
+    pub fn clear(&self, _tid: Id, _cids: &[Id]) -> MetaResult<()> {
```

Comment:
_xxx id is for unread var naming idiom in Rust. So it is not good to appear in API's parameters. I personally pickup naming like this: ```cids``` in API, ```cids0```, ```cids1```... in the body for similar but different, or ```cids_ps`` (this is ```cids``` for parts store)

Reply:
Thanks so much for your suggestion!